src/components/Project/ContentTab/QueryView.vue

```diff
@@ -1,34 +1,41 @@
 <template>
```

Comment:
You might as well create a folder for views, there will be others. ContentTab/View/Data.vue... etc

Reply:
Sure